packages/math-input/src/components/keypad/__tests__/keypad-button.test.tsx

```diff
@@ -4,15 +4,16 @@ import * as React from "react";
 import "@testing-library/jest-dom";
 
 import Keys from "../../../data/key-configs";
-import {KeypadButton} from "../keypad-page-items";
+import {KeypadButton} from "../keypad-button";
```

Comment:
I only deleted `Button.test.tsx` after confirming this file tested the same thing.